rook/README.md

```diff
@@ -0,0 +1,27 @@
+[![Docker Repository on Quay](https://quay.io/repository/cybozu/rook/status "Docker Repository on Quay")](https://quay.io/repository/cybozu/rook)
+
+Rook container
+==============
+
+This directory provides a Dockerfile to build a rook container that runs [rook][].
+
+Because of the following reasons, the Dockerfile uses [cybozu-go/rook:release][].
+1. The current version of Rook (v1.2.2) does not support partition (`part`) device type, though `master` branch already supported it. So we need to use `master` branch source code.
+2. The current version of Rook (v1.2.2) does not support `topologySpreadConstraints` feature. So we need to patch the original code of Rook.
+
+Until a release from Rook supports the above two features, the following update procedure is needed:
```

Comment:
To support the above-mentioned features, we should update this container as follows.

Reply:
I believe my sentence is preferred, so it just retained.